Source/Orts.Simulation/Common/Scripting/TrainControlSystem.cs

```diff
@@ -79,6 +80,11 @@ public abstract class TrainControlSystem : AbstractScriptClass
         /// </summary>
         public Func<bool> DoesNextNormalSignalHaveTwoAspects;
         /// <summary>
+        /// Name of Head 0 of next normal signal.
+        /// int: position of signal in the signal sequence along the train route, starting from train front; 0 for first signal;
+        /// </summary>
+        public Func<int, string> NextNormalSignalMainHeadSignalType;
```

Comment:
Any reason for this call when `NextGenericSignalFeatures` returns the same information, and more?

Reply:
The only reason is that there is already a script out using this call. But I agree deleting it from the official OR release, and will do.

Reply:
Until new calls get mainlined, we make no guarantees about their availability.

Reply:
Yes, but these calls replace the Locomotive() call, which IS mainlined and which someone wants to remove...

Reply:
Bump! This is the last blocker before I can approve.

Reply:
OK, I had forgotten this. Hook now deleted.